app/models/sk-requested-app.ts

```diff
@@ -0,0 +1,22 @@
+import { attr } from '@ember-data/model';
+
+import SkAppModel from './sk-app';
```

Comment:
add type to import

Reply:
export default class SkRequestedAppModel extends SkAppModel


SkAppModel' cannot be used as a value because it was imported using 'import type'.